### PR TITLE
Feature/Homepage-Frontend

### PR DESCRIPTION
**Pull request by Laracrofttx**

### CODE REVIEW

File: CryptoWorld.News/CryptoWorld.News.Core/Interfaces/INewsService.cs [hunk 1]
```diff
@@ -7,6 +7,8 @@ public interface INewsService
     {
         public Task <List<string>> GetNewsUrlsAsync(int pagesCount);
         public Task <List<PageNewsModel>> GetPageNewsModelAsync(List<string> urls);
+        public Task<List<FilterNewsModel>> GetAllNewsFromTheLastSevenDays();
+        public Task<List<FilterNewsModel>> GetAllNewsFromTheLastTwentyDays();
```
Comment (VelinArbov) [inline CryptoWorld.News/CryptoWorld.News.Core/Interfaces/INewsService.cs]: This methods are the same please merge them into one generic method

File: CryptoWorld.News/CryptoWorld.News.Core/Services/News/NewsService.cs [hunk 2]
```diff
@@ -142,7 +145,43 @@ private async Task<Category> GetOrCreateCategory(string categoryName)
             await dbContext.SaveChangesAsync();
 
             return category;
-        }  
+        }
+        public async Task<List<FilterNewsModel>> GetAllNewsFromTheLastSevenDays()
+        {
+            var daysAgo = DateTime.Now.AddDays(-7);
```
Comment (VelinArbov) [inline CryptoWorld.News/CryptoWorld.News.Core/Services/News/NewsService.cs]: Merge this 2 methods in one

**Review by VelinArbov — CHANGES_REQUESTED** [review]

File: CryptoWorld.News/CryptoWorld.News.Core/Services/News/NewsService.cs (hunk 2 above)
Comment (Laracrofttx) [reply CryptoWorld.News/CryptoWorld.News.Core/Services/News/NewsService.cs]: Done

File: CryptoWorld.News/CryptoWorld.News.Core/Interfaces/INewsService.cs (hunk 1 above)
Comment (Laracrofttx) [reply CryptoWorld.News/CryptoWorld.News.Core/Interfaces/INewsService.cs]: Done

**Pull request merged**